text_recognizer/data/iam_synthetic_paragraphs.py

```diff
@@ -76,13 +94,97 @@ def __repr__(self) -> str:
         )
         return basic + data
 
-    @cachedproperty
```

Comment:
I'm not sure if caching guarantees the images will be loaded only once. Need to read more!